backend/Models/Coordinates.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Internships\Models;
+
+use JsonSerializable;
+
+class Coordinates implements JsonSerializable
+{
+    protected float $latitude;
+    protected float $longitude;
+
+    public function __construct(string $coordinates)
```

Comment:
I would rather want to have a normal constructor with lat and lang. This parsing could be then moved to some other class.